spec/views/content_assets/show.html.erb_spec.rb

```diff
@@ -7,6 +7,7 @@
 
   it "renders attributes in <p>" do
```

Comment:
How does this test check that the attributes are rendered with a `<p>` tag?

Reply:
removed <p> from description